ui/src/components/admin/Model/ModelSingleHeader.js

```diff
@@ -100,20 +101,19 @@ const ModelSingleHeader = ({ modelName }) => {
   useEffect(() => {
     const refreshModelData = async () => await fetchModelData(modelName);
 
-    if (modelName && isPublishingModel(modelName) && (!publishState || !publishState.current)) {
```

Comment:
Removing the check on `publishState` here since we also need this to set back to `'publishing'` when you re-trigger a publish or publish the same model multiple times without refreshing